packages/collector/test/tracing/cloud/aws/dynamodb/util.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * (c) Copyright IBM Corp. 2021
+ * (c) Copyright Instana Inc. and contributors 2021
+ */
+
+'use strict';
+
+const AWS = require('aws-sdk');
+AWS.config.update({ region: 'us-east-2' });
+const dynamoDB = new AWS.DynamoDB();
+const interval = 1000;
+const MAX_WAIT_TIME = 10000;
+
+exports.checkTableExistence = function checkTableExistence(tableName, expectsToExist = true) {
+  let timer = 0;
+  return new Promise(resolve => {
+    const intervalId = setInterval(() => {
+      timer += interval;
+      if (timer >= MAX_WAIT_TIME) {
+        resolve(`Timeout after ${timer} ms`);
```

Comment:
Shouldn't we rather `reject` here instead of `resolve`?

Reply:
> Shouldn't we rather `reject` here instead of `resolve`?

My thinking was that even though a timeout occurred, I still would like to continue with the tests, rather than halting everything.
I can change it back if you think it makes more sense to just halt everything, assuming that the table was not created/deleted. I don't have a strong opinion here.

Reply:
Wouldn't there be follow up failures anyway if the table was not actually created? But in general, I do not have a strong opinion here.